README.md

```diff
@@ -112,6 +112,7 @@ decorator.datasource.p6spy.logging=slf4j
 decorator.datasource.p6spy.log-file=spy.log
 # Custom log format, if specified com.p6spy.engine.spy.appender.CustomLineFormat will be used with this log format
 decorator.datasource.p6spy.log-format=
+decorator.datasource.p6spy.include-parameter-values=true
```

Comment:
Let's change it to 
`decorator.datasource.p6spy.tracing.include-parameter-values=true`